content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md

```diff
@@ -13,62 +13,64 @@ weight: 80
 <!-- overview -->
 
 <!--
-This page shows how to create an External Load Balancer.
+This page shows how to create an external load balancer.
 -->
 本文展示如何创建一个外部负载均衡器。
 
-<!-- 
-This feature is only available for cloud providers or environments which support external load balancers. 
--->
-{{< note >}}
-此功能仅适用于支持外部负载均衡器的云提供商或环境。
-{{< /note >}}
-
 <!--
-When creating a service, you have the option of automatically creating a
-cloud network load balancer. This provides an externally-accessible IP address
-that sends traffic to the correct port on your cluster nodes
+When creating a {{< glossary_tooltip text="Service" term_id="service" >}}, you have
+the option of automatically creating a cloud load balancer. This provides an
+externally-accessible IP address that sends traffic to the correct port on your cluster
+nodes,
 _provided your cluster runs in a supported environment and is configured with
 the correct cloud load balancer provider package_.
 -->
-创建服务时，你可以选择自动创建云网络负载均衡器。这提供了一个外部可访问的 IP 地址，
-可将流量分配到集群节点上的正确端口上
-（ _假设集群在支持的环境中运行，并配置了正确的云负载平衡器提供商包_）。
+创建 {{< glossary_tooltip text="服务" term_id="service" >}} 时，你可以选择自动创建云网络负载均衡器。
+负载均衡器提供外部可访问的 IP 地址，可将流量发送到集群节点上的正确端口上
+（ **假设集群在支持的环境中运行，并配置了正确的云负载均衡器驱动包**）。
 
 <!--
-For information on provisioning and using an Ingress resource that can give
-services externally-reachable URLs, load balance the traffic, terminate SSL etc.,
-please check the [Ingress](/docs/concepts/services-networking/ingress/)
+You can also use an {{< glossary_tooltip term_id="ingress" >}} in place of Service.
+For more information, check the [Ingress](/docs/concepts/services-networking/ingress/)
 documentation.
 -->
-有关如何配置和使用 Ingress 资源为服务提供外部可访问的 URL、负载均衡流量、终止 SSL 等功能，
-请查看 [Ingress](/zh/docs/concepts/services-networking/ingress/) 文档。
+你还可以使用 {{< glossary_tooltip text="Ingress" term_id="ingress" >}} 代替 Service。
+更多信息，请参阅 [Ingress](/zh/docs/concepts/services-networking/ingress/) 文档。
 
 ## {{% heading "prerequisites" %}}
 
-* {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+{{< include "task-tutorial-prereqs.md" >}}
+
+<!--
+Your cluster must be running in a cloud or other environment that already has support
+for configuring external load balancers.
+-->
+你的集群必须在已经支持配置外部负载均衡器的云或其他环境中运行。
 
 <!-- steps -->
 
 <!--
-## Configuration file
+## Create a Service
+
+### Create a Service from a manifest
 
 To create an external load balancer, add the following line to your
-[service configuration file](/docs/concepts/services-networking/service/#loadbalancer):
+Service manifest:
 -->
-## 配置文件
+## 创建服务
```

Comment:
Please add a `{#anchor}` to each title.

Ref: [Localization Guild](https://kubernetes.io/zh/docs/contribute/localization_zh/#%E5%85%B3%E4%BA%8E%E9%93%BE%E6%8E%A5)

Reply:
Thanks for the review, added anchor as suggested.